core/src/tile/tileWorker.cpp

```diff
@@ -144,6 +185,10 @@ void TileWorker::setScene(std::shared_ptr<Scene>& _scene) {
     }
 }
 
+void TileWorker::notifyAll() {
+    m_condition.notify_all();
+}
+
```

Comment:
We needed this notification at some point where we were making two conditions to stop the worker thread (the initial one that is still present here; enqueue, not running) and when raster tasks were cancelled are ready. 
It is probably ok to remove those notifications now.